src/locales/strings/enUS.json

```diff
@@ -687,5 +687,7 @@
   "export_transaction_error": "Start date should be earlier than the end date",
   "export_transaction_export_error": "No transactions in the date range chosen",
   "export_transaction_loading": "Exporting Transactions…",
-  "step": "Step"
+  "step": "Step",
+  "string_scan": "Scan",
+  "string_enter": "Enter"
```

Comment:
Change string key to provide more context on the string to be translated. Use "enter_as_in_enter_address_with_keyboard"

Reply:
fixed